configure

```diff
@@ -4126,7 +4126,6 @@ printf "%s\n" "Compiler is $CC" >&6; }
 #
 # Set options for each compiler. Suffix is for WIN32
 LIBS="${LIBS} -L." #
-LDFLAGS=""
```

Comment:
configure is generated by autoconf from configure.ac.
Instead of editing in configure, you should edit in configure.ac, then run autoconf to generate the new configure.
When committing, add both configure.ac and configure to the commit

Reply:
Oh, sorry. Didn't see that augtoconf is used. (Otherwise, in this case, the configure file is often not under source control...). I have corrected it.